src/locales/en.json5

```diff
@@ -355,7 +355,7 @@
     },
     // This is shown when the Edit button is disabled.
     "editDisabled": "You cannot edit encrypted Submissions.",
-    "binaryLinkTitle": "File was submitted. Click to download.",
+    "binaryLinkTitle": "Click to download.",
```

Comment:
I think we could also consider these options:

- Click to download file.
- Click to download Submission Attachment.
- Click to download Attachment.

I think it's sort of nice to say what's being downloaded. Though "file"/"Attachment" is still pretty ambiguous.